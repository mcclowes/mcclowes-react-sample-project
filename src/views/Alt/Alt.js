import Page from "../../components/Page";
import React from "react";
import { Text } from "../../components/UIKit";

const Alt = (props) => {
  return (
    <Page>
      <Text>
        Made by <a href="https://github.com/mcclowes">@mcclowes</a>
      </Text>
    </Page>
  );
};

export default Alt;
