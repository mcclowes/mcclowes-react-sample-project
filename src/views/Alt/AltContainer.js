import React from "react";
import Alt from "./Alt";

const AltContainer = (props) => {
  return <Alt {...props} />;
};

export default AltContainer;
