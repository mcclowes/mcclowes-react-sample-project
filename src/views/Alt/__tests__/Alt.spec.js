import Alt from "../Alt";
import React from "react";
import { shallow } from "enzyme";

const defaultProps = {};

const renderComponent = (newProps) => {
  const props = {
    ...defaultProps,
    ...newProps,
  };

  return shallow(<Alt {...props} />);
};

let wrapper;

it("renders component", () => {
  wrapper = renderComponent();

  expect(wrapper).toMatchSnapshot();
});
