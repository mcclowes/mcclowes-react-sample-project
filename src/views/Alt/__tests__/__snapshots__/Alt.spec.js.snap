// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders component 1`] = `
<Page>
  <Text>
    Made by 
    <a
      href="https://github.com/mcclowes"
    >
      @mcclowes
    </a>
  </Text>
</Page>
`;
